Readme.md

```diff
@@ -1,3 +1,8 @@
+# parallel_tests
+
+[![Gem Version](https://badge.fury.io/rb/parallel_tests.svg)](https://rubygems.org/gems/parallel_tests)
+[![Build Status](https://travis-ci.org/grosser/parallel_tests.svg)](https://travis-ci.org/grosser/parallel_tests/builds)
```

Comment:
Sure, I didn't even see it down there. I'll remove it.

It looks like the timings are still not working on jruby, I'll try larger values.